spec/controllers/bgeigie_imports_controller_spec.rb

```diff
@@ -19,6 +19,10 @@
 
       it 'should get all imports' do
         expect(bgeigie_imports.size).to eq(4)
+
+        # should ordered by newly created import first
+        created_at = bgeigie_imports.map { |i| i.created_at }
```

Comment:
Looks like rubocop is calling this line out btw. Should be `map(&:created_at)` - also turned rubocop on for semaphore too ;)

Reply:
Fixed. Thx.